### ISSUE TITLE
Nadeko cannot use !clear in conjunction with !autocmdadd 

### ISSUE THREAD

**Issue by hemiipatu** [Kwoth/NadekoBot#2306]
When using `!autocmdadd X !clear`, where _X_ is the time. Nadeko fails to clear both user messages and its' own. 

I believe that this shouldn't be the case and may be a bug.

**Comment by Kwoth** [comment]
read clear help and you'll see it's not a bug.

**Comment by hemiipatu** [comment]
I have read clear help but it states nothing RE: the bot not being able to delete its' own messages. It only shows how to properly operate the command.

I have reworked the command in a number of configurations as to delete the previous message that the bot automatically posts by using `!autocmdadd`.

I have tried `!autocmdadd 120 !clear`, `!autocmdadd 120 !prune`, `!autocmdadd 120 !prune @nadeko`. All have failed to delete the message(s) previously posted by the bot and I cannot find an explanation as to why in either the bots' help message of here on Github.
